src/views/Submit.vue

```diff
@@ -145,17 +145,31 @@
 						@keydown.ctrl.enter="onKeydownCtrlEnter"
 						@update:values="(values) => onUpdate(question, values)" />
 				</ul>
-				<NcButton
-					alignment="center-reverse"
-					class="submit-button"
-					:disabled="loading"
-					native-type="submit"
-					type="primary">
-					<template #icon>
-						<NcIconSvgWrapper :svg="IconSendSvg" />
-					</template>
-					{{ t('forms', 'Submit') }}
-				</NcButton>
+				<div class="form-buttons">
+					<NcButton
+						alignment="center-reverse"
+						class="submit-button"
+						:disabled="!hasAnswers"
+						native-type="reset"
+						type="tertiary-no-background"
+						@click.prevent="showRestartSubmissionDialog = true">
+						<template #icon>
+							<NcIconSvgWrapper :svg="IconRefreshSvg" />
+						</template>
+						{{ t('forms', 'Restart submission') }}
```

Comment:
We should use the "clear form" string here as well to be more consistent.

```suggestion
						{{ t('forms', 'Clear form') }}
```